bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -33,6 +35,53 @@ WebApp.connectHandlers.use('/check', (req, res, next) => {
   res.end(JSON.stringify(payload));
 });
 
+WebApp.connectHandlers.use('/locale', (req, res) => {
+  let defaultLocale = 'en';
+  let availableTranslations = ['en', 'en-US', 'pt-BR'];
+  let defaultMessages = {};
+  let newMessages = {};
+  let languageRegion = null;
+  let language = null;
+  let foundl = false;
+  let foundlr = false;
+
+  let browserLang = req.query.locale.split('-');
+
+  if (browserLang[1]) {
+    languageRegion = browserLang[0] + '-' + browserLang[1].toUpperCase();
+    language = browserLang[0];
+  } else {
+    language = browserLang[0];
+  }
+
+  defaultMessages = Locales[defaultLocale];
```

Comment:
Lines 50-75: 
Could perhaps use .filter https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter
and .map to extract `locales` from `availableTranslations`